actioncable/lib/action_cable/channel/streams.rb

```diff
@@ -90,8 +95,11 @@ def stream_from(broadcasting, callback = nil)
       # Start streaming the pubsub queue for the <tt>model</tt> in this channel. Optionally, you can pass a
       # <tt>callback</tt> that'll be used instead of the default of just transmitting the updates straight
       # to the subscriber.
-      def stream_for(model, callback = nil)
-        stream_from(broadcasting_for([ channel_name, model ]), callback)
+      #
+      # Pass `coder: ActiveSupport::JSON` to decode messages as JSON before passing to the callback.
+      # Defaults to `coder: nil` which does no decoding, passes raw messages.
+      def stream_for(model, callback = nil, coder: nil, &block)
```

Comment:
Not sure. I meant to track down why we designed for a lambda arg vs a block before deprecating+removing.